docs/Metrics/DeployTime.md

```diff
@@ -16,11 +16,26 @@ The time it takes from when a PR is merged to when it is deployed.
 
 
 ## How is it calculated?
-You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+1. You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+2. Connect pr/mr to the right `deployment`
+![](/img/Metrics/pr-commit-deployment-relation.jpeg)
+   - Find the [commits diff](the link to refdiff plugin) between two consecutive deployments by deployments' commit_sha 
```

Comment:
The link should be changed to 'https://devlake.apache.org/docs/Plugins/refdiff/'. I used a placeholder before.

Reply:
sorry, I thought it's not ready, so i used a placer